src/jvmMain/kotlin/com/sdercolin/vlabeler/video/VideoState.kt

```diff
@@ -0,0 +1,73 @@
+package com.sdercolin.vlabeler.video
+
+import androidx.compose.runtime.getValue
+import androidx.compose.runtime.mutableStateOf
+import androidx.compose.runtime.setValue
+import androidx.compose.ui.awt.ComposeWindow
+import androidx.compose.ui.unit.Dp
+import androidx.compose.ui.unit.dp
+import com.sdercolin.vlabeler.audio.PlayerState
+import com.sdercolin.vlabeler.util.toMillisecond
+import java.awt.Container
+
+class VideoState(val debug: Boolean = true) {
+    var width: Dp by mutableStateOf(RememberWidth)
+    var height: Dp = RememberHeight
+
+    var window: ComposeWindow? = null
+    lateinit var container: Container
+
+    var playerState: PlayerState? = null
```

Comment:
Please make them `private` or `private set` if possible.

If they are only used in extension functions outside this class, consider moving those extension functions into this class as normal member functions so that they can be `private`

Reply:
I moved `window` and `container` to `NewWindowVideo.kt` and `VideoCore.kt` respectively and made them private per file. I think it is even better considering the `VideoState` class can become a muddy monolithic.